### ISSUE TITLE
if onSelect is not set and onClick is, the script breaks.

### ISSUE THREAD

**Issue by kafeltz** [missive/emoji-mart#189]
The error will be thrown when you open the emoji and then select one of them.

My example:
```
<Picker
    native={true}
    showPreview={false}
    i18n={categories}
    exclude={['search', 'recent', 'nature', 'foods', 'activity', 'places', 'objects', 'symbols', 'flags', 'custom']}
    color={'#2f3540'}
    emojiTooltip={true}
    emojisToShowFilter={emojisToShowFilter}
    onClick={this.handleClickEmoji}
    style={{ position: 'absolute', bottom: '55px', right: '33px', width: this.state.width }}
/>
```
It breaks in this file/line: https://github.com/missive/emoji-mart/blob/master/src/components/picker/nimble-picker.js#L240

To workaround this, I must set both properties:
```
onClick={this.handleClickEmoji}
onSelect={this.handleClickEmoji}
```

![image](https://user-images.githubusercontent.com/1765469/39936979-b766ca4a-5524-11e8-917b-3023fcba2a68.png)

**Comment by EtienneLem** [comment]
It’s been fixed by #183, I shall do a quick release right away.

**Comment by kafeltz** [comment]
Thx, I need it right now.
